### DISCUSSION TITLE
Is it possible to display alt text?

### DISCUSSION THREAD

**Discussion by techques** [edrlab/thorium-reader#2102] [Q&A]
I would like to have the option in Thorium to visually display an image's alt text. I cannot find this as an existing feature?

**Comment by danielweck** [comment]
Hello, this feature does not exist. Where would you suggest displaying the "alt" text? 
Bearing in mind, technically we are not allowed to inject actual DOM elements in the markup, other than at the very end of the document as this would not interfere with annotations, search, bookmarks, etc. (CSS-level content generation could work, but the reliability of a styling-only technique would depend on authored CSS which is notoriously unpredictable / inconsistent).

I assume you mean the "alt" attribute of img elements, but what about aria-label/described-by and title attributes, etc. (or SVG title element descendant, as vector graphics can be treated equally to raster images in some cases)

Thank you for your input!

**Comment by techques** [comment]
I was thinking perhaps as a foldout under the image/caption, or as a popout. However, I'm not knowledgeable enough to determine if this is allowed within the specifications. (I suppose what I'm referring to is only the "alt" attribute of img elements.)
